src/integrationTest/java/org/opensearch/security/privileges/ActionPrivilegesTest.java

```diff
@@ -314,9 +383,20 @@ public void positive_full() throws Exception {
                 assertThat(result, isAllowed());
             }
 
+            @Test
+            public void apiTokens_positive_full() throws Exception {
```

Comment:
Getting index privileges right is a quite challenging thing, as many non-obvious cases (aliases, data streams, do_not_fail_on_forbidden) must be considered. 

Therefore, I would recommend to add another dimension to the test parameters defined by the `IndexPrivileges` test. This dimension would produce a `PermissionBasedPrivilegesEvaluationContext` instead of a `RoleBasedPrivilegesEvaluationContext` using a central utility method. Then, you could use the existing test methods and be sure to have all cases covered.